### ISSUE TITLE
Adaptive card not read when activity has 'text'

### ISSUE THREAD

**Issue by froitag** [Microsoft/BotFramework-WebChat#702]
It appears the web chat does not read adaptive card attachments using text-to-speech when the activity has `text`.

Furthermore, it appears only the first adaptive card is read when there are multiple cards attached.

Tested with the adaptive card example from the [bot framework docs](https://docs.microsoft.com/en-us/bot-framework/nodejs/bot-builder-nodejs-send-rich-cards#send-an-adaptive-card) with the native browser as well as cognitive services options.

Classic question: is this a bug or a feature?

**Comment by danmarshall** [comment]
Hi @froitag - it probably is a bug. We will investigate, thanks for reporting :)

**Comment by puripuneet** [comment]
I noticed that the original title of this issue was "Adaptive card 'speak' property ignored", which is exactly the problem I'm seeing.  I had opened issue #894 to report that problem, but that issue was closed (pending my appeal to re-open it).  I'm finding that in all cases the 'speak' property in Adaptive cards is non-functional, even when the adaptive card is in response to something that the user said by first clicking on the microphone.  If there is any way to make the 'speak' property in an adaptive card work, it would be great if someone could share that.
